### DISCUSSION TITLE
[FR]实现文献自动化整理归类功能

### DISCUSSION THREAD

**Discussion by llity** [windingwind/zotero-actions-tags#126] [Ideas]
以前很习惯基于tags的管理思路，把分类目录命名为 #xx，给文章添加标签 #xxx

实现自动将添加 #xx tag 的文献移动到对应的分类目录。

希望actions-tags 在 zotero 上实现更多自动管理，自动归档、自动清理断开的附件、自动删除标记 #delete、自动清理回收站多少天的文献...

感谢

**Comment by windingwind** [comment]
插件只实现基础功能，我写的基础自定义命令仅作抛砖引玉，这些需要社区贡献
